project-base/.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,31 @@
+on: [push]
+name: "Docker build"
+jobs:
+    build-whole-application:
+        name: Build application and run standards checks and tests
+        runs-on: 'ubuntu-20.04'
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Build application
+                run: echo 1 | ./scripts/install.sh
```

Comment:
isn't possible that we start bumping into Docker limits as it's pulled anonymously?

Reply:
Yes it is - I honestly hope, that limit of 100 pulls per 6 hours will be enough. I do not see any other way right now.